src/Cassandra/RowPopulators/ExecutionInfo.cs

```diff
@@ -85,6 +86,20 @@ public IPEndPoint QueriedHost
         /// </summary>
         public ConsistencyLevel AchievedConsistency { get; private set; }
 
+        /// <summary>
+        /// After a successful schema-altering query (ex: creating a table), the driver will check if
+        /// the cluster's nodes agree on the new schema version. If not, it will keep retrying for a given
+        /// delay (configurable via <see cref="Builder.WithMaxSchemaAgreementWaitSeconds"/>).<para/>
+        /// 
```

Comment:
NIT: We can use `<para>` tags to separate paragraphs.

Reply:
Good point